_includes/components/secondarySiteNav.njk

```diff
@@ -0,0 +1,17 @@
+{%- macro secondarySiteNav(items, pageUrl) %}
+<nav id="nav-secondary" aria-labelledby="secondary-site-nav-title">
+  <h2 id="secondary-site-nav-title" hidden>About</h2>
```

Comment:
Maybe use a different accName here? And use a span instead of an h2 since it appears before the about h1.